suite-native/module-settings/src/components/ViewOnly/AboutViewOnlyBottomSheet.tsx

```diff
@@ -0,0 +1,62 @@
+import { BottomSheet, Button, TitleHeader, VStack, Text, Box } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+
+import { OrderedListItem } from './OrderedListItem';
+
+type AboutViewOnlyBottomSheetProps = {
+    isVisible: boolean;
+    onClose: () => void;
+};
+
+export const AboutViewOnlyBottomSheet = ({ isVisible, onClose }: AboutViewOnlyBottomSheetProps) => {
+    let order = 1;
+
+    if (!isVisible) {
+        return null;
+    }
+
+    return (
+        <BottomSheet isVisible={isVisible} onClose={onClose} isCloseDisplayed={false}>
+            <VStack spacing="large" paddingHorizontal="small">
+                <TitleHeader
+                    title={<Translation id="moduleSettings.viewOnly.about.title" />}
+                    subtitle={<Translation id="moduleSettings.viewOnly.about.subtitle" />}
+                />
+                <VStack>
+                    <Text variant="callout">
+                        <Translation id="moduleSettings.viewOnly.about.contentTitle" />
+                    </Text>
+                    <Box>
+                        <Translation
+                            id="moduleSettings.viewOnly.about.content"
+                            values={{
```

Comment:
Why not like this:

```tsx
        <Translation
            id={keyPath}
            values={{
                li: chunks =>
                    chunks.map(
                        (row, index) =>
                            row && (
                                <BulletListItem key={`${row}`} variant="hint" color="textSubdued">
                                    {row}
                                </BulletListItem>
                            ),
                    ),
            }}
        />
```

You have index available here

Reply:
I tried that index is always 0

Reply:
b410c0a136

Reply:
chunks is an array of 1 item... 😞

Reply:
Exactly. This was the first thing I wanted to use and discarded for this reason.